app/src/main/res/layout/item_download.xml

```diff
@@ -1,106 +1,103 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.google.android.material.card.MaterialCardView
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
-  android:layout_margin="@dimen/card_margin"
-  xmlns:android="http://schemas.android.com/apk/res/android">
+  android:layout_margin="@dimen/card_margin">
 
-<LinearLayout
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:minHeight="?android:attr/listPreferredItemHeight"
-  android:orientation="horizontal"
-  android:paddingStart="@dimen/activity_horizontal_margin"
-  android:paddingTop="@dimen/activity_vertical_margin"
-  android:paddingEnd="@dimen/activity_horizontal_margin">
-
-  <ImageView
-    android:id="@+id/libraryDownloadFavicon"
-    android:layout_width="@dimen/favicon_width"
+  <LinearLayout xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
```

Comment:
Just general clean-up or part of the Lollipop change?

Reply:
Hide white space changes before reviewing.
This was just fixing incorrect indentations.

Reply:
Didn't know that existed. Cool!

Reply:
In that case this looks good!